src/XMakeTasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -3166,6 +3167,22 @@ Copyright (C) Microsoft Corporation. All rights reserved.
 
   <!--
     ============================================================
+                                        _GenerateCompilerDependencyFile
+
+    Generate file with a list of compile source files. This is used to track
+    incremental build between build execution specifically around addition and
+    deletions of file globs (e.g. <Compile Include="**\*.cs" />).
+    ============================================================
+    -->
+  <Target Name="_GenerateCompilerDependencyFile" BeforeTargets="CoreCompile">
+    <WriteLinesToFile Lines="@(Compile->'%(FullPath)')" File="$(IntermediateOutputPath)Compile.deps" Overwrite="True" WriteOnlyWhenDifferent="True" />
```

Comment:
I don't like `.deps`, especially since there's already the `.deps.json` files for .NET Core. Maybe `.CompilerInputs`?